module/EntraBeta/config/ModuleMetadata.json

```diff
@@ -32,5 +32,7 @@
     ],
     "releaseNotes": "See https://github.com/microsoftgraph/entra-powershell.",
     "version": "0.20.0",
-    "Prerelease": "preview"
+    "Prerelease": "preview",
+    "dotNetVersion":"4.7.2",
+    "powershellVersion":"5.1"
```

Comment:
```suggestion
    "powerShellVersion":"5.1"
```

Reply:
No one spells it like "powerShellVersion".
Please, don't do that.